tests/kubeless-test-client/svcbind.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  namespace: kubeless-test
+  name: redis
+spec:
+  clusterServiceClassExternalName: redis
+  clusterServicePlanExternalName: micro
+  parameters:
+    redisPassword: VXU3ZVRkY2tVdlFVMkNHYlYwM2gK
+    resources:
+      requests:
+        memory: 96Mi
+---
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceBinding
+metadata:
+  namespace: kubeless-test
+  name: redis-binding
+spec:
+  instanceRef:
+    name: redis
+---
+apiVersion: gateway.kyma.cx/v1alpha2
+kind: Api
```

Comment:
Do you really need to expose your service? Is it called from outside of cluster?

Reply:
Yes. In the test we want to ensure that the kubeless controller creates the function correctly and that it is available via api/ingress. This is short-lived and removed at the end of the test.